### DISCUSSION TITLE
how to hide some column from "set columns"

### DISCUSSION THREAD

**Discussion by kstang** [bosskmk/pluto_grid#526] [Q&A]
as titled , pls find the image below , I want to hide the first column form the list , which the key is '' : 

I created the dummy column that contain the button for edit and delete row

<img width="693" alt="Screenshot 2022-07-24 at 6 28 41 PM" src="https://user-images.githubusercontent.com/8650432/180643047-0a8ec71c-452c-4b8d-9ef3-574c5509b843.png">

**Comment by bosskmk** [comment]
@kstang 
There is no function to hide a specific column in the column setting list.

**Reply by kstang** [reply]
possible to add the that feature at future release ? should be a common use case to group the button in the unnamed column

**Reply by darkomenx** [reply]
This is a very good idea...

**Comment by Jullies** [comment]
I discovered it is possible to hide Column by just setting the width to 0, it worked for me

PlutoColumn(
        title: 'ID',
        field: 'test',
        type: PlutoColumnType.text(),
        width: 0),
